### PR TITLE
Update label explanations

### PR DESCRIPTION
**Pull request by arturs105**
Squash whenever

### CODE REVIEW

File: github-workflow.md [hunk 1]
```diff
@@ -36,13 +36,17 @@ Unless an issue is invalidated otherwise, it should only be closed by merging a
 
 We use labels sparingly to give further information to issues. Here are the common labels we currently use, and when to use them.
 
+- `blocking` - the issue or pull request bolcks some or all work for someone. In other words: someone is waiting for this issue or pull request to be resolved
```
Comment (amulware) [inline github-workflow.md]: I'm not sure there is agreement on this. Specifically, the point that anybody can add the label to issues/prs that block any of their work. Comments from others would be welcome.

File: github-workflow.md [hunk 2]
```diff
@@ -36,13 +36,17 @@ Unless an issue is invalidated otherwise, it should only be closed by merging a
...
-- `glitch` - a non-functional issue in the app, usually related to misbehaving UI
 - `cleanup` - deals with refactoring, or other changes that should not change how the app functions
-- `test-week` - this issue can be worked on by someone with little experience with the app's intricacies
-- `review-pending` - the issue has a pull request, which is awaiting review
+- `critical` - the issue or pull request has the highest priority
+- `glitch` - a non-functional issue in the app, usually related to misbehaving UI
 - `needs-design` - the issue still needs UI/UX design
 - `on-hold` - the issue cannot be resolved at the moment, or is suspected to be invalid and awaits further testing
+- `pending-review` - the issue has a pull request, which is awaiting review
+- `snoozed` - the issue was moved from previous week's milestone to the next weeks milestone, so it has a higher priority than the rest
```
Comment (amulware) [inline github-workflow.md]: `weeks` -> `week's`

File: github-workflow.md (hunk 1 above)
Comment (Odrakir) [reply github-workflow.md]: I don't use that label at all, I rather go get someone through slack :)

Comment (amulware) [reply github-workflow.md]: Wait until you have a bigger team. :D

Comment (Odrakir) [reply github-workflow.md]: More people to bother

Comment (amulware) [reply github-workflow.md]: After internal discussion, let's change it to 'the issue or pull request blocks other work in the current milestone' (also note there is a typo in the current text)

File: github-workflow.md [hunk 3]
```diff
@@ -36,13 +36,18 @@ Unless an issue is invalidated otherwise, it should only be closed by merging a
...
 - `cleanup` - deals with refactoring, or other changes that should not change how the app functions
-- `test-week` - this issue can be worked on by someone with little experience with the app's intricacies
-- `review-pending` - the issue has a pull request, which is awaiting review
+- `crash` - a serious issue that crashes the application
+- `critical` - the issue or pull request has the highest priority
+- `glitch` - a non-functional issue in the app, usually related to misbehaving UI
 - `needs-design` - the issue still needs UI/UX design
 - `on-hold` - the issue cannot be resolved at the moment, or is suspected to be invalid and awaits further testing
+- `pending-review` - the issue has a pull request, which is awaiting review
+- `snoozed` - the issue was moved from previous week's milestone to the next week's milestone, so it has a higher priority than the rest
```
Comment (willsb) [inline github-workflow.md]: This is kinda confusing. Should be `from previous week's to current week's`

File: github-workflow.md [hunk 4]
```diff
@@ -36,13 +36,18 @@ Unless an issue is invalidated otherwise, it should only be closed by merging a
...
+- `test-week` - this issue can be worked on by someone with little experience with the app's intricacies
```
Comment (willsb) [inline github-workflow.md]: This is the exact definition of [`up-for-grabs`](http://up-for-grabs.net/#/) and I suggest we rename it

Comment (amulware) [reply github-workflow.md]: I'm fine with changing the description (pr welcome), but I think `test-week` makes a lot of sense for our use case of having test weeks.

File: github-workflow.md (hunk 3 above)
Comment (amulware) [reply github-workflow.md]: :+1: PR welcome :D

**Pull request merged**